widget/index.js

```diff
@@ -55,7 +55,21 @@ var WidgetGenerator = yeoman.generators.Base.extend({
       }
       // TODO: validate not empty string?
     },
-    // TODO: checkbox prompt for 2D/3D once the WAB supports 3D
+    {
+      type: 'checkbox',
+      message: 'Is it 2D or 3D?',
+      name: 'widgetType',
+      choices: [
+        {
+          value: 'is2d',
+          name: '2D'
+        },
+        {
+          value: 'is3d',
+          name: '3D'
+        }
+      ]
+    },
```

Comment:
Is it valid to have a widget that is *both* 2d and 3d? Either way, we should add a validator to this prompt that ensures that at least one is checked, and possibly not allow both to be checked (in case that is not valid)